### PR TITLE
Feature save email toggle

### PR DESCRIPTION
**Pull request by webatspeed**

### CODE REVIEW

File: src/main/resources/db/changelog/changeset/0025_add_notify_enquiries_repeating/add-notify-enquiries-repeating.sql [hunk 1]
```diff
@@ -0,0 +1,2 @@
+alter table userservice.consultant
+    add notify_enquiries_repeating bit default true not null after encourage_2fa;
```
Comment (hill-daniel) [inline src/main/resources/db/changelog/changeset/0025_add_notify_enquiries_repeating/add-notify-enquiries-repeating.sql:2]: Will we add more email notification settings in the future? If so, we would have to add an additional column for each setting.

Comment (webatspeed) [reply src/main/resources/db/changelog/changeset/0025_add_notify_enquiries_repeating/add-notify-enquiries-repeating.sql:2]: Yes, we would. What's the problem? :) A list-esque data type wouldn't even satisfy second normal form.

Comment (hill-daniel) [reply src/main/resources/db/changelog/changeset/0025_add_notify_enquiries_repeating/add-notify-enquiries-repeating.sql:2]: I was thinking of an extra table for all the settings we accumulate for the user

Comment (webatspeed) [reply src/main/resources/db/changelog/changeset/0025_add_notify_enquiries_repeating/add-notify-enquiries-repeating.sql:2]: Which would have a 1-1 connection to either the consultant table or the user table. 1-1 connections are denormalizations. The either is a complication nobody wants.

**Review by hill-daniel — APPROVED** [review]

**Pull request merged**